### PR TITLE
Siri Hemsley - Portfolio

### PR DESCRIPTION
**Pull request by sirimatsdotter**
Hi!

I need to start with saying that I was sick most of this week and had a very hard time with completing this project. So I am handing in this today knowing that I have not passed the blue requirements. 

I'm happy I was able to produce _something_, but after testing with WAVE I have seen there is still somethings that is not accessible for everyone. 

I also realised that my solution with having project description and link appear on hover might not be ideal for screensreaders and have not yet added any extra code to make it accessible. 

**The following is things that I know I need to do, that I have not had time for yet:**
- Decrease image sizes (since there are taking long time to load right now)
- Make compatible with screen readers, especially on the project cards with the text on hover. 
- Check browser compatibility for Safari, Edge, Firefox

### CODE REVIEW

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
You didn't provide a link to the hosted version of your site. Please do it ASAP to make me possible to test it.

**Comment by sirimatsdotter** [comment]
Sorry, it seems like my readme has not been added correctly (which had the link)

Find it here https://clever-knuth-74ee12.netlify.app/

@KarolCislowski

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300;500;700&display=swap" rel="stylesheet">
+<script src="https://kit.fontawesome.com/e58657a886.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header class="hero">
+    <picture>
+      <!-- <source srcset="./assets/hero-img-mobile.jpg" media="(max-width: 767px)"/>
+      <source srcset="./assets/hero-img-tablet.jpg" media="(max-width: 1199px)"/> -->
+      <img class="hero-img" src="./assets/hero.jpg" alt="" />
```
Comment (KarolCislowski) [inline index.html:18]: All your images are very high resolution so it makes your page to load very slowly. You can use photoshop or apps like https://tinypng.com/ to make them smaller.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <p>ES6</p>
+        <p>HTML</p>
+        <p>CSS</p>
+        <p>Git</p>
+        <p>GitHub</p>
+        <p>npm</p>
+        <p>Flexbox</p>
+      </div>
+    </section>
+
```
Comment (KarolCislowski) [inline index.html:45]: I think it would be nice to have some Contact section on your portfolio 🤓

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+<body>
+  <header class="hero">
+    <picture>
+      <!-- <source srcset="./assets/hero-img-mobile.jpg" media="(max-width: 767px)"/>
+      <source srcset="./assets/hero-img-tablet.jpg" media="(max-width: 1199px)"/> -->
+      <img class="hero-img" src="./assets/hero.jpg" alt="" />
+    </picture>
+    <h1 class="hero-title">SIRI HEMSLEY <br> ━ Technigo Bootcamp Portfolio</h1>
+  </header>
+
```
Comment (KarolCislowski) [inline index.html:22]: You have some margin between your header and the rest of the page. I think you should get rid of it.

**Review by KarolCislowski — APPROVED** [review]
It is a very nice portfolio page 🦄  
I left below a shortlist of fixes to make it even better. Please make them soon 😸

**Pull request closed without merging**